### ISSUE TITLE
vertical_parameter_alignment shouldn't validate parentheses

### ISSUE THREAD

**Issue by marcelofabri** [realm/SwiftLint#1042]
This example shouldn't trigger `vertical_parameter_alignment` rule:

```swift
func validateFunction(
  _ file: File, kind: SwiftDeclarationKind,
  dictionary: [String: SourceKitRepresentable]
) -> [StyleViolation] {
  // hello
}
```
